nnpdf_data/nnpdf_data/new_commondata/STAR_2009_2JET_MIDRAP_200GEV/metadata.yaml

```diff
@@ -0,0 +1,84 @@
+setname: "STAR_2009_2JET_MIDRAP_200GEV"
+
+nnpdf_metadata:
+  experiment: "STAR"
+  nnpdf31_process: "DIJET"
+
+arXiv:
+  url: https://arxiv.org/pdf/1610.06616
+iNSPIRE:
+  url: "https://inspirehep.net/literature/1493842"
+hepdata:
+  url: "https://www.hepdata.net/record/ins1493842"
+  version: 1
+
+version: 1
+version_comment: "Initial implementation"
+
+implemented_observables:
+  - observable:
+      {
+        description: "ALL as function of $M_{inv}$, OS",
+        label: "$A_{LL}$",
+        units: "",
+      }
+    observable_name: OS-ALL
+    process_type: DIJET_POL
+    ndata: 7
+    tables: [9]
+    kinematics:
+      variables:
+        m_jj: { description: "dijet mass", label: "$m_{jj}$", units: "GeV" }
+        sqrts:
+          {
+            description: "center of mass energy",
+            label: r"$\sqrt(s)$",
+            units: "GeV",
+          }
+      file: kinematics_OS.yaml
+    kinematic_coverage: [m_jj, sqrts]
```

Comment:
Same here. In order to compute the $(x, Q^2)$ kinematic plots, dijets require the following $(y_{(\star)}, \sqrt{s}, m_{jj})$.